content/docs/studio/how-it-works.md

```diff
@@ -1,48 +1,54 @@
 # How Iterative Studio Works
 
-[Iterative Studio](https://studio.iterative.ai/) works with the data, metrics,
-hyperparameters and model metadata that you add to your ML project Git
-repositories. It works very closely with your Git ecosystem.
+[Iterative Studio](https://studio.iterative.ai/) works very closely with your
+Git ecosystem. It parses the data, metrics, hyperparameters and model metadata
+that you add to your ML project Git repositories, and renders them on a web UI.
+It also parses and displays live metrics and plots sent by [DVCLive]; these
+don't need to be pushed to the Git repositories.
 
-This video illustrates how Iterative Studio works closely with your Git
-ecosystem.
+The following video illustrate how Iterative Studio works with data from your ML
+projects and experiments.
 
 > Note that we have renamed DVC Studio to Iterative Studio and Views to
 > Projects.
 
 https://www.youtube.com/watch?v=5xM5az78Lrg
```

Comment:
The video and it's purpose (being in this place of the doc) should be self-explanatory. Also, proper admonition:

```suggestion
https://www.youtube.com/watch?v=5xM5az78Lrg

<admon>

We have renamed DVC Studio to Iterative Studio and Views to Projects.

</admon>
```

Reply:
Am not making this change, coz we use this same format in 3 places where we have videos; this note applies to the videos specifically.